twitter_bot.py

```diff
@@ -39,12 +44,19 @@ def nawab_read_list(self):
             search_list.append(row["Proto_list"])
         return (search_list)
 
-    def nawab_store_id(self, tweet_id):
+    def nawab_store_id(self, tweet_id, isrelevant):
         ### Store a tweet id in a file
-        dicts = { 'Date_time' : [time.strftime("%m/%d/%Y %I:%M:%S %p ")],
-                'Id': [str(tweet_id)]}
-        data = pd.DataFrame(dicts)
-        data.to_csv(self.dirpath + 'tid_store.csv',mode='a',header=False,index=False)
+        if isrelevant:
+            dicts = { 'Date_time' : [time.strftime("%m/%d/%Y %I:%M:%S %p ")],
+                    'Id': [str(tweet_id)]}
+            data = pd.DataFrame(dicts)
+            data.to_csv(self.dirpath + 'tid_store.csv',mode='a',header=False,index=False)
+        else:
+            dicts = { 'Date_time' : [time.strftime("%m/%d/%Y %I:%M:%S %p ")],
+                    'Id': [str(tweet_id)]}
+            data = pd.DataFrame(dicts)
+            data.to_csv(self.dirpath + 'backup_tid_store.csv',mode='a',header=False,index=False)
+            
```

Comment:
At a code stand point looks good to me!

While you can separate this into a completely new function.